### DISCUSSION TITLE
How to send Youtube video?

### DISCUSSION THREAD

**Discussion by sreekanthmr** [salman0ansari/whatsapp-api-nodejs#221] [Q&A] [answered]
I'm able to send mp4 videos through mediaurl end point but I was not successful in sending Youtube videos.

**Comment by salman0ansari** [comment, marked as answer]
add yt-dl middleware 
detect yt url
if true
await till video download
then send video using media endpoint

**Reply by sreekanthmr** [reply]
@salman0ansari Thank you for your suggestion.  I'll check it out.
